auth-web/src/services/org.services.ts

```diff
@@ -70,8 +70,8 @@ export default class OrgService {
     return axios.put(`${ConfigHelper.getAuthAPIUrl()}/orgs/${orgId}`, createRequestBody)
   }
 
-  static async getAffiliatiatedEntities (orgIdentifier: number): Promise<AxiosResponse<Businesses>> {
-    return axios.get(`${ConfigHelper.getAuthAPIUrl()}/orgs/${orgIdentifier}/affiliations`)
+  static async getAffiliatiatedEntities (orgIdentifier: number): Promise<AxiosResponse<any>> {
```

Comment:
would be good to still have the type here if possible

Reply:
I can't keep the type here, because the api response changed, and I can't modify too much on type of Businesses, cause it will affect other components other than affiliations.